tests/suites/ck/ck.sh

```diff
@@ -3,14 +3,31 @@
 run_deploy_ck() {
 	echo
 
-	local name model_name file overlay_path kube_home
+	local name model_name file overlay_path overlay_url kube_home
 	name="deploy-ck"
 	model_name="${name}"
 	file="${TEST_DIR}/${model_name}.log"
 
 	ensure "${model_name}" "${file}"
 
-	overlay_path="./tests/suites/ck/overlay/${BOOTSTRAP_PROVIDER}.yaml"
+	case "${BOOTSTRAP_PROVIDER:-}" in
+	"ec2")
+		overlay_url="https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/overlays/aws-storage-overlay.yaml"
+		;;
+	"gce")
+		overlay_url="https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/overlays/gcp-storage-overlay.yaml"
+		;;
+	"azure")
+		overlay_url="https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/overlays/azure-cloud-overlay.yaml"
+		;;
```

Comment:
I can see why you want to do this, but it does feel wrong that we're calling out to a specific location. If they ever change this, we're broken. 

This will be a fun one to debug in the future.

Reply:
Not much we can do here. It's better that it breaks here, than breaking because of incompatibility with the charmhub delivered bundle.